webplugin/js/app/km-richtext-markup-1.0.js

```diff
@@ -249,11 +249,9 @@ Kommunicate.markup = {
     },
     getQuickRepliesTemplate: function () {
         return `
-        <div class='km-cta-multi-button-container' >
```

Comment:
This css might be getting used for different css customizations by customer
@Prochnost55

Reply:
Sir this css has been already appended to the quick reply container and here it is redundant.
so it can be removed